python/packages/nisar/workflows/focus.py

```diff
@@ -1111,7 +1111,7 @@ def process_rfi(cfg: Struct, raw_data: np.ndarray,
         rfi_likelihood = isce3.signal.rfi_freq_null.run_freq_notch(
             raw_data,
             opt_fnf.num_pulses_az,
-            num_rng_blks=opt.num_range_blocks,
+            num_samples_rng_blk=opt.num_samples_rng_blk,
```

Comment:
The `isce3.signal.rfi_freq_null.run_freq_notch()` doesn't currently take a `num_samples_rng_blk` parameter, so this won't work.

https://github.com/isce-framework/isce3/blob/6093f479bca52ba4c903f403e9f853dfee2fb754/python/packages/isce3/signal/rfi_freq_null.py#L61-L75

Also, `opt` doesn't have a `num_samples_rng_blk` attribute.

Did you mean to do this instead?

```suggestion
            num_rng_blks=opt_fnf.num_range_blocks,
```